cves/CVE-2013-4315.yml

```diff
@@ -66,9 +70,18 @@ fixes_vcc_instructions: |
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
 - commit: 87d2750b39f6f2d54b7047225521a44dcd37e896
-  note:
+  note: |
+     [1.4.x] Prevented arbitrary file inclusion with {% ssi %} tag and relative paths.
+     Thanks Rainer Koirikivi for the report and draft patch.
+     This is a security fix; disclosure to follow shortly.
+     Backport of 7fe5b656c9 from master
 - commit: 988b61c550d798f9a66d17ee0511fb7a9a7f33ca
-  note:
+  note: |
+    [1.5.x] Prevented arbitrary file inclusion with {% ssi %} tag and relative paths.
+    Thanks Rainer Koirikivi for the report and draft patch.
+    This is a security fix; disclosure to follow shortly.
+    Backport of 7fe5b656c9 from master
+
 vccs:
 - commit: 4e8864a882201e8ff99085ad0e53171fc028c9c2
   note: This VCC was discovered automatically via archeogit.
```

Comment:
Maybe some notes on what's going on for the VCCs?

Reply:
Added developer notes to these VCCs.